src/TreeViewList/TreeViewList.tsx

```diff
@@ -130,10 +204,21 @@ const TooltipTreeItem = (
     children?: React.ReactNode;
   }
 ) => {
+  // check if a node has children
+  const hasChildren = (children: React.ReactNode) => {
+    return React.Children.count(children) > 0;
+  };
+
   return (
     <Tooltip
       disableFocusListener
-      title={props.tooltip ? <>{props.tooltip}</> : ""}
+      title={
+        props.tooltip && !hasChildren(props.children) ? (
+          <>{props.tooltip}</>
+        ) : (
+          ""
+        )
+      }
```

Comment:
Not sure I agree with this change. If you only want tooltip for children in a specific application then don't provide tooltip prop for the parents. Don't think this logic should live in the component as it already has an optional interface to set tooltip on each node.